packages/loader/container-loader/src/container.ts

```diff
@@ -789,8 +790,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     public async attach(request: IRequest): Promise<void> {
-        assert(this.loaded, 0x0d4 /* "not loaded" */);
-        assert(!this.closed, 0x0d5 /* "closed" */);
+        if (this.loaded) {
+            throw new UsageError("Container must be loaded to attach");
```

Comment:
error codes should be camel case. which is still human readable, but more machine reable, also this can even be part of a user facing error due to lack of localization, to make it a phase isn't any better than camel case